frontend/pages/hosts/details/HostDetailsPage/HostActionsDropdown/helpers.tsx

```diff
@@ -336,7 +332,21 @@ const setOptionsAsDisabled = (
     );
   }
 
-  if (!hostScriptsEnabled) {
+  if (hostScriptsEnabled === null) {
```

Comment:
Please add this edge case to `HostActionsDropdown.tests.tsx` under the `describe("Run script action", () => {`

Reply:
yes, thanks. waiting until final product decisions on what to do before adding tests.